interface/wx/stdpaths.h

```diff
@@ -79,6 +79,18 @@ class wxStandardPaths
         */
         Dir_Cache,
 
+        /**
+            Directory containing configuration information.
+
+            Example return values:
+            - Unix: `~/.config`
+            - Windows: `C:\Users\username\AppData\Roaming`
```

Comment:
Should there be a Local version of this (for Windows) like there is for userdatadir? (to allow windows users to keep this in appdata\local)

Reply:
Update: Though looking at wxFileConfig closer, I notice it does not support Local, so this may not be too important.

Reply:
`GetUserDir(Dir_Config)` returns the same directory as `GetUserConfigDir()` under Windows as I think this makes most sense.